### PR TITLE
chore: add stETH rewards message

### LINKED ISSUE #1213: Add info that stETH includes the rebase (positive) from staking rewards
**Is your feature request related to a problem? Please describe.**
There is number of questions from the community: 'If I use stETH as collateral in Aave, can I still get rewards given by stETH?'. Answer is yes. This info is hard to find.

**Describe the solution you'd like**
Add info box and an faq [link](https://blog.lido.fi/aave-integrates-lidos-steth-as-collateral/) on the Reserve page.

**Describe alternatives you've considered**
—

### PR DESCRIPTION
**Pull request by 0x4Graham**
## General Changes

- Fixes #1213 

## Developer Notes

Add any notes here that may be helpful for reviewers.

## Author Checklist

- [x]  The base branch is set to `main`
- [x]  The title is using [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) formatting
- [x]  The Github issue has been linked to the PR in the Development section
- [x]  The General Changes section has been filled out
- [ ]  Developer Notes have been added (optional)

## Reviewer Checklist

- [ ]  End-to-end tests are passing without any errors
- [x]  Code style generally follows existing patterns
- [x]  Code changes do not significantly increase the application bundle size
- [x]  New third-party packages, if any, do not introduce potential security threats
- [x]  There are no CI changes, or they have been OK’d by the devops team
- [ ]  Code changes have been quality checked in the ephemeral URL
- [ ]  QA verification has been completed
- [ ]  There are two or more approvals from the core team
- [ ]  Squash and merge has been checked

### CODE REVIEW

File: src/modules/reserve-overview/ReserveConfiguration.tsx [hunk 1]
```diff
@@ -305,6 +305,23 @@ export const ReserveConfiguration: React.FC<ReserveConfigurationProps> = ({ rese
               )}
             </Box>
           )}
+          {reserve.symbol == 'stETH' && (
+            <Box>
+              <Warning severity="info">
+                <AlertTitle>Staking Rewards</AlertTitle>
```
Comment (drewcook) [inline src/modules/reserve-overview/ReserveConfiguration.tsx:311]: I think this may also need to be wrapped in `<Trans>` tags as well.

**Review by drewcook — CHANGES_REQUESTED** [review]
Looks good! Just one small change.

This shows that we may want to have our `Warning` component support these titles if we ever had other warnings that look like this one.  We could do that easily.  This works for now though

File: src/modules/reserve-overview/ReserveConfiguration.tsx (hunk 1 above)
Comment (0x4Graham) [reply src/modules/reserve-overview/ReserveConfiguration.tsx:311]: Added in trans. 

Would need help with the warning component

**Review by drewcook — DISMISSED** [review]
No need to improve the Warning component at this point.  Just making a mental note for myself. LGTM 👍

**Review by defispartan — DISMISSED** [review]
LGTM

**Review by defispartan — APPROVED** [review]

**Review by grothem — APPROVED** [review]

**Pull request merged**